Sources/_RegexParser/Regex/Parse/Diagnostics.swift

```diff
@@ -173,6 +174,8 @@ extension ParseError: CustomStringConvertible {
       return "expected ASCII for '\(c)'"
     case .expectedCustomCharacterClassMembers:
       return "expected custom character class members"
+    case .dotNetSubtractionMustBeLast:
+      return "character class members may not appear after .NET subtraction"
```

Comment:
Do we actually want to tell users something about `.NET` here? Diagnostics should probably be phrased in terms of Swift regex syntax and violations there-of.